### ISSUE TITLE
Add "free access until" field to articles.

### ISSUE THREAD

**Issue by caseyg** [caseyg/americanethnologist#45]

**Comment by daveleins8** [comment]
Got more information, below is the email from our publisher's about when content is free:

"Thanks for checking in on this and for refreshing my memory about the arrangement we made for switching the free sample issue this year.  Currently, the free sample issue is the Feb 2017 issue (AE 44:1).  Normally, this would remain as the free sample issue through 2017 and we would update this once the Feb 2018 issue publishes.  This year, we’ve agreed to set the May 2017 issue free once that publishes.
 
There is no hard date in our systems for when the free sample issue will expire.  When the May 2017 issue publishes online, please let me know and I will have this set as the free sample, replacing the Feb 2017 issue.  At that point, you could also loop in the web master for your site so that you can make the update there, as well.  When the Feb 2018 issue publishes, we can do the same.
 
In addition to the free sample issue, we sometimes set individual articles free for 30 days for promotional purposes, but I don’t think we’ve done this in the past couple of months.  Usually, this would be for new content, but articles in the current issue are already set as free samples.
 
I hope this answers your questions, but if you have others just let me know!"

**Comment by daveleins8** [comment]
Higher priority on this enhancement please.

**Comment by caseyg** [comment]
Dave, can you help me understand what would be a good way to indicate which articles are free, based on the info from the publisher?

I thought a "free until X date" field on each article might make sense, but it sounds like it's more of an issue-wide thing which changes quarterly, so maybe better suited to a text field on the /read page, announcing the current sample issue?

**Comment by daveleins8** [comment]
Yes, you're right about that. It is free across an entire issue. There
isn't an easy way of knowing when these issues are made free or not, it's
basically ad hoc when the board asks the publisher if it can be, and for
how long. Announcing on the read page will work well. I do think in the
past there were articles here and there that are free within an issue that
isn't, but right now that isn't the case. I'll ask Caitrin to get a solid
answer.

**Comment by daveleins8** [comment]
FYI

Currently using

<div style="text-align: center;"><span style="display: inline-block; padding: 5px; border: 2px solid #ff0000; font-size: 90%; color: #ff0000;"> Open access article </span></div>

to show individual articles as open access in TOC.
